### DISCUSSION TITLE
Winmerge silence background

### DISCUSSION THREAD

**Discussion by wachtw** [WinMerge/winmerge#1105] [Ideas]
Can it be possible to use winmerge completely silent on the background using the command line in a batch file and creating the output to a standard text file?
Option /x and /xq runs the program silently, but the outcoming windows are present on the screen.

**Comment by sdottaka** [comment]
Unfortunately, there is currently no way to run WinMerge completely in the background. However, you can use the `-minimize` and `-noninteractive` options to hide the window, although it will appear in the taskbar momentarily.

~~~
"c:\Program Files\WinMerge\WinMergeU.exe" -noninteractive -minimize file1\t1.txt  file2\t1.txt -or c:\tmp\report.html
~~~
